posthog/api/test/batch_exports/test_create.py

```diff
@@ -109,6 +110,13 @@ def test_create_batch_export_with_interval_schedule(client: HttpClient, interval
         assert args["batch_export_id"] == data["id"]
         assert args["interval"] == interval
 
+        if interval == "hour":
+            assert batch_export.jitter == dt.timedelta(minutes=15)
+        elif interval == "day":
+            assert batch_export.jitter == dt.timedelta(hours=1)
+        elif interval == "every 5 minutes":
+            assert batch_export.jitter == dt.timedelta(minutes=1)
```

Comment:
not sure if this one will pass since it's actually 1.25 minutes?

Reply:
Ah yeah. I'll make it 1 minute as I don't think the .25 is too critical and 1 minute is more round. The values are pretty arbitrary anyways.